sru/release-27.3/ua-messaging-disabled.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+series=$1
+name="test-systemd-$series"
+DEB_PATH=$2
+
+lxc launch ubuntu-daily:$series $name
+sleep 5
+lxc file push $DEB_PATH $name/tmp/ua.deb
+echo "Disabling ua-messaging.timer"
```

Comment:
Can we add a `list-timers` call before disabling just to easily see it before the disable?
```suggestion
echo "List timers in default state"
echo "--------------------------"
lxc exec $name -- sudo systemctl list-timers --all
echo "--------------------------"
echo "Disabling ua-messaging.timer"
```